packages/block-editor/src/components/list-view/index.js

```diff
@@ -71,6 +72,7 @@ function ListViewComponent(
 		isExpanded = false,
 		showAppender = false,
 		blockSettingsMenu: BlockSettingsMenu = BlockSettingsDropdown,
+		parentId = null,
```

Comment:
Maybe if `parentId` is set we should use that to work out what is in `blocks`?

Reply:
Sounds like a good idea.

The `blocks` prop may need to be deprecated.

Reply:
I'm not sure. In some ways I'd rather keep it as it makes the component more flexible, although we don't use that prop at the moment,